axelor-sale/src/main/resources/domains/AppSale.xml

```diff
@@ -45,6 +45,7 @@
       help="This configuration enables you to activate an editable grid on quotation / sales order lines, allowing you to quickly add and complete quotation lines from the grid view."/>
     <boolean name="isDiscountEnabledOnEditableGrid"
       title="Enable discount on editable lines"/>
+    <boolean name="enableCartManagement" title="Cart Management"/>
```

Comment:
Rename isCartManagementEnabled to respect name convention.
Change title to 'Enable cart management'